versioned_docs/version-v22/execution.md

```diff
@@ -74,8 +74,6 @@ Here is the code for the standard behaviour.
 ```java
 public class SimpleDataFetcherExceptionHandler implements DataFetcherExceptionHandler {
 
-    private static final Logger logNotSafe = LogKit.getNotPrivacySafeLogger(SimpleDataFetcherExceptionHandler.class);
```

Comment:
The diff looks strange because the documentation evolves rather than being completely written. In this case this line getting removed corresponds to SLF4J being removed as a dependency